src/components/fields/localized-multiline-text-field/localized-multiline-text-field.spec.js

```diff
@@ -23,16 +23,20 @@ class Story extends React.Component {
     value: { en: '', de: '' },
     selectedLanguage: 'en',
   };
+
   state = {
     value: this.props.value,
   };
+
   handleChange = event => {
+    const value = event.target.value;
     if (this.props.onChange) this.props.onChange(event);
+    event.persist();
```

Comment:
Should this happen before invokving the onChange too?

Reply:
moved in [80554ad22bbd8c658662270c0ba6462112734454](https://github.com/commercetools/ui-kit/pull/1016/commits/80554ad22bbd8c658662270c0ba6462112734454)